javascript/rollup/rollup.config.template.js

```diff
@@ -32,7 +33,7 @@ if (isBinBuild) {
     bundle({
       plugins: [esbuild(), styles({
         modules: true,
-      }), json()],
+      }), json(), image()],
```

Comment:
Probably not the best pattern to just keep adding plugins here instead of making it a bit more configurable, thoughts?

Reply:
Yeah for the less common use-cases (I would image image is one of them), we could either add config options to contruct/add plugins, or just leave it to the individual package to run their own rollup build

Reply:
I remember I had added the `json` plugin for something one-off as well, though I can't recall the exact use case, maybe for content bundling or something. I can revert this PR and incorporate image loading rollup config directly in the devtools Bazel build.